src/map/npc.hpp

```diff
@@ -1532,13 +1532,34 @@ enum e_job_types
 	JT_4_CLB_SS_AJ,
 	JT_4_CLB_SS_LK,
 	JT_4_SMART_ANDRE,
-
-	JT_4_ALEXSANDER = 10601,
+	JT_4_ALEXSANDER,
```

Comment:
Did you make sure the value stays the same here? Is JT_4_SMART_ANDRE 10600 then?

Reply:
Edit: Verified myself, seems good.